cmd/git-pair/main.go

```diff
@@ -17,6 +19,13 @@ func main() {
 	app := &cli.App{
 		Version: diagnostics.Version,
 		Usage:   "A tool to make it easier for git based pairing for co-authoring commits",
+		Before: func(cCtx *cli.Context) error {
+			if !git.IsGitRepo() {
+				return errors.New("Not executed from a git repository")
```

Comment:
When raised it yields

```
malachi@pulsar ~ $ git-pair 
2023/05/30 21:39:36 Not executed from a git repository
```

Any idea how we can format the error handling to just dump the error message?